bankbot.lic

```diff
@@ -0,0 +1,229 @@
+# want_script_output
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#bankbot
+=end
+
+custom_require.call(%w(common common-money common-validation drinfomon events))
+
+class Bankbot
+  include DRC
+  include DRCM
+
+  def initialize
+    # TODO: Write transactions to a file
+
+    @settings = get_settings
+    @last_action = Time.now
+    @bankbot_debug = UserVars.bankbot_debug
+    @denominations = ["Kronar", "Lirum", "Dokora"]
+
+    @banker_file_name = "bot_banker_#{checkname}.dat"
+    UserVars.bot_bank = UserVars.bot_bank || Hash.new{|hash, key| hash[key] = Hash.new}
+  end
+
+  def help(character)
+    @last_action = Time.now
+    fput "whisper #{character} Supported commands are:"
+    fput "whisper #{character}   (help) - this list"
+    fput "whisper #{character}   (bal|balance) - view your current bot balance"
+    fput "whisper #{character}   (with|withdraw) [amount] [denomination] - withdraw [amount] [denomination] from your bot balance."
+    fput "whisper #{character} I accept coins via TIPs to hold for you."
+  end
+
+  def balance(character)
+    @last_action = Time.now
+    unless has_bot_account(character)
```

Comment:
Yes, I don't see the harm. At least a harm that would really warrant the rewrite/refactor at this junction.